test/decl/var/property_wrappers.swift

```diff
@@ -1723,3 +1723,56 @@ struct TestConcrete1 {
     // ...
   }
 }
+
+// SR-11477
+
+// Two initializers that can default initialize the wrapper //
+
+@propertyWrapper
+struct SR_11477_W1 { // Okay, because we'll pick the best one here
```

Comment:
Do we have a test that verifies which initializer is picked by writing `@SR_11477_W1 var foo: Int`?

Reply:
No, but I have added one now!